.gitignore

```diff
@@ -109,3 +109,6 @@ story-dependencies.png
 # Build/test byproducts
 cache/
 .hypothesis
+
+# VS Code configuration
```

Comment:
Sorry if this is a bit of a drive-by. Surely I can't be the only one using vs code?

Reply:
You can set this as part of the global gitignore for your system. In general, detritus created by the project should be excluded by the project gitignore, and detritus created by the user's tooling/operating system etc should be excluded by the user's gitignore.

This should be just be as simple as creating a file in `~/.config/git/ignore`, but there are some more details here if that's helpful:
https://git-scm.com/docs/gitignore

Reply:
Ideal, shall do thanks!